ansible/roles/gpu/tasks/nvidia-gpu-RHEL8.yaml

```diff
@@ -1,24 +1,30 @@
 ---
-# If possible we use headers from main OS repo for this kernel.
-- name: Install Kernel Header and Devel from OS repo for Current Kernel
-  dnf:
-      name:
-        - "kernel-headers-{{ hostvars[inventory_hostname].ansible_kernel }}"
-        - "kernel-devel-{{ hostvars[inventory_hostname].ansible_kernel }}"
-
-- name: import epel repository key
```

Comment:
let me get it straight

the new AMI in the PR is bringing in the headers for 8.2 an for 8.4 we are installing via nvidia script?

Reply:
No, the headers are not brought in by the AMI. The kernel headers are generally to be installed. Our problem was for the RHEL8.2.1 contained kernel `4.18.0-193.65.2.el8_2.x86_64` there was no header package available from RH.

We updated the query to now match the original RHEL8.2 variant bringing us `4.18.0-193.el8` for which there are headers available.

RHEL 8.4 in the variant we currently base on has kernel headers available - no issue there. They get installed via that same step which is exactly right.